grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -717,6 +771,41 @@ static GrpcStatusFunction toGrpcStatusFunction(
         return interceptors;
     }
 
+    private void collectDecorators(Class<?> clazz,
+                                   @Nullable String path,
+                                   ServerServiceDefinition serverServiceDefinition) {
+        final String serviceName =
+                path == null ? serverServiceDefinition.getServiceDescriptor().getName() : path;
```

Comment:
Should we flip the condition for readability because `path` has a higher priority?
 
```suggestion
        final String serviceName =
                path != null ? path : serverServiceDefinition.getServiceDescriptor().getName();
```